### DISCUSSION TITLE
torchscript

### DISCUSSION THREAD

**Discussion by paulelenord** [ACEsuit/mace#41] [Q&A] [answered]
How to compile model to torchscript and then deploy in ase? (If yet available)

**Comment by davkovacs** [comment, marked as answer]
Please for now use the torchscript_develop branch. We are planning to merge that probably in the next release.

**Reply by davkovacs** [reply]
You can use the models in ASE without compiling them though. It should work on the main branch, and the develop branch has several new features including stresses enabling even NPT dynamics in ASE.
